src/serac/physics/solid_mechanics.hpp

```diff
@@ -255,13 +241,11 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
     v_.SetSize(true_size);
 
     du_.SetSize(true_size);
-    du_ = 0.0;
-
     dr_.SetSize(true_size);
-    dr_ = 0.0;
-
     predicted_displacement_.SetSize(true_size);
-    predicted_displacement_ = 0.0;
+
+    shape_displacement_ = 0.0;
```

Comment:
Do we consider resetting the shape displacement not part of the initialization step? In other words, do we expect this to be reset if you call the `reset` command? Any thoughts @kswartz92 ?

Reply:
I've gone back and forth on this, but I think it makes sense to separate resetting the design (i.e. parameters/shape) from resetting of the states (and related transient information e.g. time steps). If a user was solving an inverse problem for initial conditions, they may want to set parameters/shape and then leave them during their inverse problem. Similarly, I sometimes use "parameter" fields to define field quantities that are not actually changing in a design optimization. Having a reset states only method allows me to not place those fields on a lido graph. It does make a "reset" method slightly more confusing since it doesn't get the solver back to exactly the same place as a newly constructed solver, but I think the tradeoff is worth it.

Reply:
keeping separate for now.  We have a resetStates, can add a resetDesign , or resetDesignFields (or whatever) when it is needed, which may be never as the user can already essentially do this, I think.